templates/includes/featured_highlight_gallery.html

```diff
@@ -0,0 +1,38 @@
+{% load wagtailimages_tags wagtailcore_tags i18n %}
+
+<!--TODO: Boilerplate HTML added. This is not the final product, work to be done by FE dev-->
+<div class="container mt-4" data-container-name="{{ highlight_gallery.title }}">
+    <h2>Explore records about <a href='{{ highlight_gallery.url }}'>{{highlight_gallery.title}}</a></h2>
+    <ul class="card-group--list-style-none">
+        {% for item in page.featured_highlight_gallery_items %}
```

Comment:
Maybe we just forget the `page` and this method and pull things straight from `highlight_gallery`? That might make it easier to reuse this template elsewhere in future

Reply:
In fact, maybe a `HighlightGalleryPage.items()` method would be good here, which could make use of `select_related()` and `prefetch_related()` to efficiently fetch the highlights, images and renditions in as few queries as possible.

Reply:
Hi @ababic, cheers for the feedback :)

> In fact, maybe a `HighlightGalleryPage.items()` method would be good here, which could make use of `select_related()` and `prefetch_related()` to efficiently fetch the highlights, images and renditions in as few queries as possible.

Is this to replace the current return value of `featured_highlight_gallery_items`?

Reply:
Yes :)

Reply:
Hey @ababic , I have been thinking about this one - this is using the `highlights()` function you created on the `HighlightsGalleryPage`, which returns `self.page_highlights.exclude(image__isnull=True).select_related("image").prefetch_related("image__renditions")`, is this not sufficient? Or is there a better way I can access these values in the HTML?

Reply:
Oh right. I forgot about that. If that's there already, let's just call that directly.... The `featured_highlight_gallery_items` cached property just complicates things unnecessarily.

Reply:
Should be good now! @ababic